src/lib/components/PreTestTimer.svelte

```
@@ -0,0 +1,135 @@
+<script lang="ts">
+	import { enhance } from '$app/forms';
```

ESLint

[error] 'enhance' is defined but never used.

(@typescript-eslint/no-unused-vars)